force-app/main/default/customMetadata/Bucket.ServiceStatusActive.md-meta.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomMetadata xmlns="http://soap.sforce.com/2006/04/metadata" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <label>Service Status: Active</label>
```

Comment:
IIRC the last time we added a filter we did not include any bucketed values and then in the code said if no bucketed values then include everything. This way the customer could still opt in. 
The concern is if an existing customer is not using the status field or representing Active with Active, it'd be a breaking change.